modules/virt-example-vm-node-placement-node-affinity.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+[id="virt-example-vm-node-placement-node-affinity_{context}"]
+= Example: VM node placement with node affinity
+
+In this example,
```

Comment:
In this example, VMs can only be scheduled on nodes that have a label of `example.io/example-key = example-value-1` or `example.io/example-key = example-value-2`. The constraint is met if only one of the two labels is present.

Additionally, the scheduler will prefer to avoid nodes that have a label of `example-node-label-key = example-node-label-value`, however if all nodes have this label, it will be ignored for the purposes of scheduling.